Project-Descriptions-and-Plans/CVE20/CVE20-PA-2024.md

```diff
@@ -0,0 +1,311 @@
+
+### Potential future enhancements for future project phases
+- Zce static code size reduction opcode extension
+- Supporting 2 pin compressed JTAG (CJTAG) debug interface
+- Investigation of a "tiny FPU " implementation
+ - Targeted at sensor computations at the edge 
+ - Having FP would be useful for these and other computations
+- Low granularity Physical Memory Protection (PMP) module
+
+## Who would make use of OpenHW output
+
+Companies developing microcontroller-based embedded (sub)systems or devices.
+
+
+
+## (*For Cores Projects*) Part Numbers
+*Write down the CORE-V part number assigned for each of the output cores produced/verified by this project plan*
+
+| Core produced/verified in this project		    | Part Number			| Comment				| 
+| --------------------- 							| --------------------- | ---------------------	| 
+| CV32E20											|	 CV32E20			| The project a single configuration, CV32E20, from the CVE2 repo | 
+
+## (*For Cores Projects*) TRL target
+
+*Write down the TRL target for each of the output cores produced/verified by this project plan*
+
+| Core produced/verified in this project		    | TRL Target			| Comment				| 
+| --------------------- 							| --------------------- | ---------------------	| 
+| CV32E20										    |	5					| The target for CV32E20 is TRL5		| 
+
+## Release plan
+
+*Write down the anticipated release number and release schedule for the work covered by this project plan. For example, Release 1.0.0 in 2024-12-01*
+
+| Release number		    | Planned date						| Description															| 
+| --------------------- 	| --------------------- 			| ---------------------													| 
+| CV32E20 1.0.0						| 2025-12-01						| Initial fully verified TRL5 release 				|
+
+*Add any comment about release strategy or timing such as follow-on releases *
+ 
+
+
+
+## Project deliverables (high level)
+
+*The high level project deliverables (e.g. code, verification, user document, and other deliverables) should be listed. I.e. what are the open-source deliverables? Don't list detailed features here.*
+*The following table is provided as an example*
+
+| Deliverable		    | Description			| Technical Project Leader		| 
+| --------------------- | --------------------- | ---------------------	| 
+| Item 1	    			| RTL code				|	Mike Thompson					|
+| Item 2	    			| Spike Reference Model	|	Mario Rodriguez				|
+| Item 3	    			| Verification of RTL	|	Mike Thompson					|
+| Item 4	    			| User Manual			|	Duncan Bees					|
+
+*All major deliverables require a Technical Project Leader.*
+
+## Feature List 
+
+
+*This section provides the detailed feature list of the project.*
+*If the features are outlined in detail in the PL, or within a User Manual or other document, refer to that document. If not, the feature list should be provided here*
+
+Detailed features are available at https://docs.openhwgroup.org/projects/cve2-user-manual/en/latest/01_specification/index.html
```

Comment:
Do we need to have a link here?  Eventually it will break.  Maybe just reference the "CV32E20 User Manual".

Reply:
Actually I'll remove it entirely as we recognized the user manual is not yet in good shape to portray the features and that THIS document contains the feature list